content/docs/command-reference/run.md

```diff
@@ -73,23 +73,34 @@ $ dvc run -n printer -d write.sh -o pages ./write.sh
 $ dvc run -n scanner -d read.sh -d pages -o signed.pdf ./read.sh
 ```
 
+Stage dependencies can be any file in the workspace, either untracked, or much
+more commonly, tracked by DVC or Git. Outputs will be tracked and
+<abbr>cached</abbr> by DVC when the stage is run. Every output version will be
+cached when the stage is reproduced (see also `dvc gc`).
+
 Relevant notes:
 
-- Typically, scripts being run (or a directory containing the source code) are
-  included among the specified `-d` dependencies. This ensures that when the
-  source code changes, DVC knows that the stage needs to be reproduced. (You can
-  chose whether to do this.)
+- Typically, scripts being run (or possibly a directory containing the source
+  code) are included among the specified `-d` dependencies. This ensures that
+  when the source code changes, DVC knows that the stage needs to be reproduced.
+  (You can chose whether to do this.)
 
 - `dvc run` checks the dependency graph integrity before creating a new stage.
-  For example: two stage cannot explicitly specify the same output, there should
-  be no cycles, etc.
+  For example: two stage cannot specify the same output or overlapping output
+  paths, there should be no cycles, etc.
 
 - DVC does not feed dependency files to the command being run. The program will
   have to read by itself the files specified with `-d`.
 
+- Entire directories can be tracked as outputs, which produces a single `.dir`
```

Comment:
produced as an output?

Reply:
Yes. But "tracked" is a key concept here I think. I went for "Entire directories produced by the stage can be tracked as outputs by DVC" in c02e81e2.